cdapython/results/factories/collect_result.py

```diff
@@ -59,6 +60,17 @@ def get_all(
         page_size: Union[None, int] = None,
         show_bar: bool = True,
     ) -> Result:
+        """
+        This will page through the query results automatically looping and appending result objects to a List
+        for the user.
+        Args:
+            output (str, optional): if you add this output string "full_list"or"full_df" You will either get a list or a DataFrame. Defaults to "".
+            page_size (int, optional): this can increase the page size results . Defaults to None.
```

Comment:
```suggestion
            page_size (int, optional): this can increase the page size results . Defaults to None.  Known issue: the first page is always 100, even when the size is adjusted.
```